docs/content/preview/yugabyte-platform/alerts-monitoring/universe-logging.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Export PostgreSQL logs from YugabyteDB Anywhere universes
```

Comment:
"PostgreSQL logs" -> shouldn't we say YSQL audit logs? Same question for the subtitle here in YBM / Aeon https://deploy-preview-23605--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-cloud/cloud-monitor/logging-export/